arduino-ide-extension/src/electron-main/theia/electron-main-application.ts

```diff
@@ -777,3 +842,7 @@ function updateAppInfo(
   });
   return toUpdate;
 }
+
+function isAddressInfo(arg: unknown): arg is Pick<AddressInfo, 'port'> {
+  return isObject<AddressInfo>(arg) && typeof arg.port === 'number'; // `family` might be
```

Comment:
Is the `` `family` might be `` comment incomplete? It seems quite cryptic to me (but maybe not so for the target reader?).

Reply:
Haha. Thank you! The `family` might be _"missing"_ is missing.

Reply:
https://github.com/arduino/arduino-ide/pull/2236/commits/8927b2b441f764304fc315a3d80336f3cfd13777